assets/js/theme/product/reviews.js

```diff
@@ -12,38 +12,39 @@ export default class {
         });
 
         this.$reviewsContent = $('#product-reviews');
+        this.$reviewsContentList = $('#productReviews-content', this.$reviewsContent);
         this.$collapsible = $('[data-collapsible]', this.$reviewsContent);
 
         this.initLinkBind();
         this.injectPaginationLink();
-        this.collapseReviews();
+        this.setupReviews();
     }
 
     /**
      * On initial page load, the user clicks on "(12 Reviews)" link
      * The browser jumps to the review page and should expand the reviews section
      */
     initLinkBind() {
-        const $content = $('#productReviews-content', this.$reviewsContent);
-
-        $('#productReview_link').on('click', () => {
-            $('.productView-reviewTabLink').trigger('click');
-            if (!$content.hasClass('is-open')) {
-                this.$collapsible.trigger(CollapsibleEvents.click);
-            }
-        });
+        $('#productReview_link').on('click', () => this.showReviews());
     }
 
-    collapseReviews() {
+    setupReviews() {
         // We're in paginating state, do not collapse
         if (window.location.hash && window.location.hash.indexOf('#product-reviews') === 0) {
-            return;
+            return this.showReviews();
```

Comment:
Could we put return after statement call for the reason it does not return anything

Reply:
And it should not return anything. It was combined with `return` to avoid unnecessary line - in case we in `if` parent function must stop its execution

Reply:
btw, comment above says  "... do not collapse" and we execute `this.showReviews()`. Is it correct or we need remove this comment?

Reply:
Yes, it is correct. But I have updated it to make it more precise in context of new implementation

Reply:
It is a good practice to put return empty if function should return nothing